### PR TITLE
Helm - Make license optional

### PR DESCRIPTION
**Pull request by alebabai**
## what
make license optional

### CODE REVIEW

**Comment by alebabai** [comment]
@mumoshu could your look into it, please.

File: examples/chart/teleport/values.yaml [hunk 1]
```diff
@@ -118,6 +118,11 @@ proxy:
     # We assume TLS is terminated in front of the proxy by default
     enabled: true
 
+license:
+  enabled: true
```
Comment (mumoshu) [inline examples/chart/teleport/values.yaml]: A comment here that says "set false to run Teleport in Community edition would be helpful"

**Review by mumoshu — APPROVED** [review]
Thanks for the fix! Just one thing, and LGTM overall.

We should now hold on until one of maintainers to approve this. I'm the original author of this chart but not a maintainer :)

File: examples/chart/teleport/values.yaml (hunk 1 above)
Comment (alebabai) [reply examples/chart/teleport/values.yaml]: sure, thanks

**Comment by klizhentas** [comment]
ok to test

**Pull request merged**